src/tools/genpolicy/genpolicy-settings.json

```diff
@@ -219,6 +219,8 @@
         "dns_label": "[a-zA-Z0-9_\\.\\-]+",
```

Comment:
We may want to refine this regex. Per https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names , a dns label name does not allows upper case, nor `.`, and we could limit it to 63 characters